src/reducer/video-reducer.js

```diff
@@ -0,0 +1,71 @@
+import {
+  addNewPlayList,
+  addToLikes,
+  addToWatchedVideos,
+  addToPlayList,
+  isLiked,
+  removeFromLikes,
+  removeFromPlayList,
+  increaseLikes,
+  decreaseLikes,
+} from "../utils/video-context";
+export const reducer = (state, {type, payload}) => {
+  switch (type) {
+    case "SET_DATA":
+      console.log(payload.data);
+      return {
+        ...state,
+        data: payload.data,
+      };
+    case "SET_LIKEDVIDEOS":
```

Comment:
Action type should be changed to `SET_LIKED_VIDEOS`. Same for action type below this as well.